src/manage_nsfs/manage_nsfs_events_utils.js

```diff
@@ -0,0 +1,331 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const dbg = require('../util/debug_module')(__filename);
+const _ = require('lodash');
+
+
+const EVENT_ERROR_CODE = {
+    NOOBAA: {
+        FORK_EXIT: {
```

Comment:
I think it'll be nicer to have the same format we have in errors and responses, WDYT?
ManageCLIEvent.FORK_EXIT = Object.freeze({
    event_code: 'noobaa_fork_exit',
    entity_type: 'NODE',
    event_type: 'STATE_CHANGE',
    message: 'Noobaa fork exited',
    description: 'Noobaa fork exited due to internal error',
    scope: 'NODE',
    severity: 'ERROR',
    state: 'DEGRADED',
});
also, it'll make our life easier on the mapping so we won't need the error_from, WDYT?

Reply:
@romayalon `Object.freeze()` objects are not mutatable. right? For most of the events, we need to update the message, description, and arguments. I am not sure we can do that with `Object.freeze()`.